internal/api/integrations/k8s/k8s_test.go

```diff
@@ -0,0 +1,87 @@
+package k8s
+
+import (
+	"testing"
+
+	"ship-it/internal"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAnnotationFor(t *testing.T) {
```

Comment:
Let's write these table-driven tests using named sub-tests, like so: https://github.com/Wattpad/ship-it/blob/master/internal/image_test.go#L9

I personally prefer to define named type, rather than an inline anonymous struct type, but it's fairly subjective.